lib/pprof/pprof.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pprof
+
+import (
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type PprofPort int
+
+const basePort = 6060
+
+const (
+	VchInitPort PprofPort = iota
+	ImagecPort
```

Comment:
@hickeng requested this be added for imagec in the event that it were to run into an infinite loop and/or hang.  We do not expect imagec to generate any logs, but if it is behaving abnormally, it may.